samples/nrf9160/modem_shell/Kconfig

```diff
@@ -84,6 +84,13 @@ config MOSH_REST
 	help
 	  REST client for sending and receiving simple REST requests/responses.
 
+config MOSH_CLOUD
+	bool "MQTT connection to nRFCloud"
+	default n
```

Comment:
Remove this line as it's default. I've been instructed to remove these few times although I like that it documents that it's intentionally OFF by default.

Reply:
Already commented by @tokangas